demos/src/css/chartAuthoring-demo.css

```diff
@@ -0,0 +1,25 @@
+.floe-ca-pieChart-pie {
+    float: left;
+    width: 400px;
```

Comment:
In preparation of integrating UI Options into this demo, which is what we usually do for all our websites, "em" or "rem" instead of "px" should be used for HTML element sizes. So that elements are responsive to UIO's text size change.

https://wiki.fluidproject.org/display/fluid/How+to+make+your+page+more+responsive#Howtomakeyourpagemoreresponsive-Useemsfortextsize

Same to line 7.